swagger-spec/collection_submissions/list.yaml

```diff
@@ -0,0 +1,72 @@
+# /collection_submissions/{collection_submissions_id}/actions/
+get:
+  summary: Get a list of Collection Submission Actions for a Collection Submission
+  description: >-
+
+    This returns a list of the actions that have changed the Collection Submission moderation state.
+
+    #### Permissions
+
+    This information is only available to an admin on the Collection in question.
```

Comment:
So I just double-checked this in an incognito window and I was able to open https://api.staging.osf.io/v2/collection_submissions/z9cr4-3vpkf/actions/ ?? Is this information we are showing publicly?

Reply:
The actual code is"
```
        if obj.collection.is_public:
            return True
        else:
            is_moderator = auth.user and auth.user.has_perm('view_submissions', obj.collection.provider)
            return obj.guid.referent.has_permission(auth.user, READ) or is_moderator
``` 
so it matters if you're a public collection, or a moderator or superuser.